boards/beagle/beagleconnect_freedom/board.yml

```diff
@@ -3,4 +3,12 @@ board:
   full_name: BeagleConnect Freedom
   vendor: beagle
   socs:
-  - name: cc1352p7
+    - name: cc1352p
+    - name: cc1352p7
+  revision:
+    format: custom
+    exact: true
+    default: "C7"
```

Comment:
I'm looking at this in the summary of changes, so we are good with including a default of "C7"? If so, I'll approve. If it breaks building via `west build -b beagleconnect_freedom`, I'm going to reject. I want to have "C5" support in mainline, but it isn't worth breaking default builds.